test/go-tests/test_utils.go

```diff
@@ -225,6 +228,93 @@ func CreateProject(projectName string, shipyardFilePath string, recreateIfAlread
 	return "", err
 }
 
+func CreateProjectWithSSH(projectName string, shipyardFilePath string, recreateIfAlreadyThere bool) (string, error) {
```

Comment:
Quite some stuff in `CreateProject` and `CreateProjectWithSSH` are identical - pls refactor

Reply:
done